src/promptflow/promptflow/_sdk/operations/_run_submitter.py

```diff
@@ -296,6 +297,16 @@ def _submit_bulk_run(self, flow: Flow, run: Run, local_storage: LocalStorageOper
         run._dump()  # pylint: disable=protected-access
         try:
             bulk_result = flow_executor.exec_bulk(mapped_inputs, run_id=run_id)
+            # Filter the failed line result
+            failed_line_result = \
+                [result for result in bulk_result.line_results if result.run_info.status == Status.Failed]
+            if failed_line_result:
+                # Log warning message when there are failed line run in bulk run.
+                error_log = f"{len(failed_line_result)} run(s) failed in bulk run."
```

Comment:
"x out of y runs failed in bulk run."
In which x means failed runs, y means total runs.

Reply:
update warning log